src/rules/functionality.ts

```diff
@@ -53,6 +55,8 @@ export default {
   'no-null-keyword': false,
   // Disallows shadowing variable declarations.
   'no-shadowed-variable': true,
+  // Forbids array literals to contain missing elements.
+  'no-sparse-arrays': false,
```

Comment:
The rationale behind it is `Missing elements are probably an accidentally duplicated comma.`